dask/dataframe/io.py

```diff
@@ -338,8 +338,11 @@ def from_pandas(data, npartitions, sort=True):
     ----------
     df : pandas.DataFrame or pandas.Series
         The DataFrame/Series with which to construct a dask DataFrame/Series
-    npartitions : int
+    npartitions : int (defaults to None)
```

Comment:
Yeah, I've been sloppy about following numpydoc explicitly in the past.  We
should endeavor to do this in the future though.  I'll add a TODO for
myself to go through an clean up existing docstrings.

On Thu, Mar 10, 2016 at 7:44 AM, thequackdaddy <notifications@github.com>
wrote:

> In dask/dataframe/io.py
> <https://github.com/dask/dask/pull/1035#discussion_r55698588>:
>
> > @@ -338,8 +338,11 @@ def from_pandas(data, npartitions, sort=True):
> >      ----------
> >      df : pandas.DataFrame or pandas.Series
> >          The DataFrame/Series with which to construct a dask DataFrame/Series
> > -    npartitions : int
> > +    npartitions : int (defaults to None)
>
> I will change this. Note that the parenthesis format is elsewhere in the
> io fiie as well, namely in the from_bcolz function.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dask/dask/pull/1035/files#r55698588>.
>